src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -154,13 +154,33 @@ public string UniqueId
         /// <summary>
         /// Get the Element Pinned status
         /// </summary>
-        public bool? IsPinned
+        public bool IsPinned
         {
             get 
-            { 
-                return this.InternalElement?.Pinned; 
+            {
+                if (InternalElement == null)
+                    return false;
+                return this.InternalElement.Pinned; 
             }
         }
+        
+        /// <summary>
+        /// Checks if two elements are joined
+        /// </summary>
+        /// <param name="otherElement">Element to check</param>
+        /// <returns>True if the two elements are joined, False otherwise</returns>
+        public bool IsJoined(Element otherElement)
```

Comment:
we should call this `AreJoined` since the node will have 2 `Element` inputs when it's displayed to the user.